homeassistant/components/climate/wink.py

```diff
@@ -16,11 +17,18 @@
     TEMP_CELSIUS, STATE_ON,
     STATE_OFF, STATE_UNKNOWN)
 
+_LOGGER = logging.getLogger(__name__)
+
 DEPENDENCIES = ['wink']
 
 STATE_AUX = 'aux'
```

Comment:
We should not define our own states, this makes it impossible to create a UI for it. Please use only standardized states from `climate/__init__.py`

Reply:
Hmm okay, I thought these were dynamic in the dropdown for climate devices. So I will need to add more to __init__.py the water_heater states don't match up with the current states.

Reply:
This still needs to addressed

Reply:
Can you explain what needs addressed/why?

I have tested this and it works? All if the "custom" states show up in the mode selection drop down and selecting them changes the devices state.

Reply:
We define the set of available states in the climate component, and those are the ones that should be used. Think of this as an Enum in Java.

If you don't use them you will:

 - make the Wink component a blocker for making Home Assistant translatable.
 - limit what we can do in the frontend because we can no longer assume we know which states to expect.